server/devdocs/bin/generate-proptypes-index.js

```diff
@@ -30,6 +30,16 @@ const camelCaseToSlug = ( name ) => {
 		.replace( /^-/, '' );
 };
 
+const slugToCamelCase = name => {
+	if ( ! name ) {
+		return name;
+	}
+
+	return name
```

Comment:
I wonder whether lodash's [camelCase](https://lodash.com/docs/4.17.4#camelCase) and [upperFirst](https://lodash.com/docs/4.17.4#upperFirst) suffice here?